seattle_example/seattle_tasks.yml

```diff
@@ -33,7 +33,9 @@
 #        - { src: maryland_blkgrps, dest: '' }
 
 - name: Copy geo data to server
-  copy: src=seattle_example/{{ item.src }} dest=/opt/piecewise_web/{{ item.dest }}
+  copy:
+    src: seattle_example/{{ item.src }}
+    dest: /opt/piecewise_web/{{ item.dest }}
   with_items:
     - { src: seattle_census10_blockgroups.topojson, dest: seattle_census10_blockgroups.topojson }
```

Comment:
`dest` seems to be redundant here (identical to `src`) and `dest` below is an empty string.  Could this be simplified with:

```
copy:
  src: seattle_example/{{ item }}
  dest: /opt/piecewise_web/{{ item }}
with_items:
  - seattle_census10_blockgroups.topojson
  - seattle_blkgrpce10
```